<!-- Run the aruco_detect node -->
<launch>
  <!-- namespace for camera input -->
  <arg name="camera" default="/camera"/>
  <arg name="image" default="compressed"/>
  <arg name="fiducial_len" default="0.14"/>

  <node pkg="aruco_detect" name="aruco_detect"
    type="aruco_detect" output="screen" respawn="false">
    <param name="image_transport" value="compressed" />
    <param name="publish_images" value="true" />
    <param name="fiducial_len" value="$(arg fiducial_len)"/>
    <remap from="aruco_detect/camera/compressed" to="$(arg camera)/$(arg image)"/>
    <remap from="aruco_detect/camera_info" to="$(arg camera)/camera_info"/>
  </node>
</launch>
